media-video/droidcam/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM 'http://www.gentoo.org/dtd/metadata.dtd'>
+<pkgmetadata>
+	<maintainer type="person">
+		<name>Henrik Pihl</name>
+		<email>ahvenas@gmail.com</email>
+	</maintainer>
```

Comment:
I think you would also have to include `proxy-maint` project in package metadata below your name and e-mail (https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml)